action.yml

```diff
@@ -7,14 +7,14 @@ inputs:
     description: >-
       The name that the package should be given. This is used as the `--name`
       argument to `raco pkg install`.
-    required: true
+    required: false
```

Comment:
This should probably still be required, since it defaults to the directory name which is whatever folder the repo gets cloned as.

Reply:
what's the name used for? is it not taken from the `info.rkt` file?

Reply:
(The assumption is that using the repo name as the package name is a reasonable default.)

Reply:
It shows up in `raco pkg show` and is used in `raco test -p`, `raco setup -p`, and some other random places.

The repo name is sometimes a reasonable default but people tend to make repos named like `someuser/racket-foo` or `someuser/rkt-foo` containing a package that they publish on the catalog under the name `foo`, so I like to ask people to be explicit about whether they want the repo name to be used as the package name.

Reply:
can we just read it from `info.rkt`? i'll investigate

Reply:
oh fuck yeah: [`get-info/full`](https://docs.racket-lang.org/raco/getinfo.html#%28def._%28%28lib._setup%2Fgetinfo..rkt%29._get-info%2Ffull%29%29)

Reply:
Package names aren't in `info.rkt` files, alas.